src/commands/CmdInfo.cpp

```diff
@@ -116,9 +158,27 @@ int CmdInfo::execute(std::string& output) {
     auto description = task.get("description");
     auto indent = Context::getContext().config.getInteger("indent.annotation");
 
-    for (auto& anno : task.getAnnotations())
-      description += '\n' + std::string(indent, ' ') +
-                     Datetime(anno.first.substr(11)).toString(dateformatanno) + ' ' + anno.second;
+    {
+      const size_t termWidth = Context::getContext().getWidth();
+      const size_t approxNameCol = 20;  // conservative; covers labels like "Last modified"
+      const size_t valueWidth = termWidth > approxNameCol ? termWidth - approxNameCol : termWidth;
+
+      for (const auto& anno : task.getAnnotations()) {
+        // first line looks exactly like before: <indent><timestamp><space>
```

Comment:
"before" won't mean much to a reader outside the context of this PR -- rephrase to the context of the code

```suggestion
        // first line looks like <indent><timestamp><space>
```